dotcom-rendering/src/web/components/SignInGate/gateDesigns/SignInGateCopyTestJan2023.tsx

```diff
@@ -0,0 +1,153 @@
+import { cmp } from '@guardian/consent-management-platform';
+import { Button, Link, LinkButton } from '@guardian/source-react-components';
+import { trackLink } from '../componentEventTracking';
+import type { SignInGateProps } from '../types';
+import {
+	actionButtons,
+	bodyBold,
+	bodySeparator,
+	bodyText,
+	faq,
+	firstParagraphOverlay,
+	headingStyles,
+	hideElementsCss,
+	laterButton,
+	privacyLink,
+	registerButton,
+	signInGateContainer,
+	signInHeader,
+	signInLink,
+} from './shared';
+
+export const SignInGateCopyTestJan2023 = ({
+	signInUrl,
+	guUrl,
+	dismissGate,
+	abTest,
+	ophanComponentId,
+	isMandatory = false,
+}: SignInGateProps) => {
+	const getCopyTextFromVariant: (abVariant: string | undefined) => string = (
+		abVariant,
+	) => {
+		switch (abVariant) {
+			case 'quick-and-easy': {
+				return "Register: it's quick and easy";
```

Comment:
Welcome back! 👋 

Could we use proper apostrophe in `it's` (should be `it’s).

Reply:
Hey, thank you 😊 

Oooh good spot, thank you. I've updated.